tests/integration/tier1/destructive/firewalld-config-ol8/test_firewalld_config_ol8.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import pytest
+
+from envparse import env
+
+from convert2rhel.actions.system_checks.check_firewalld_availability import FIREWALLD_CONFIG_FILE
+
+
+@pytest.mark.test_firewalld_config
+def test_firewalld_config_ol8(shell, convert2rhel):
+    """
+    Verify that properly changing the firewalld configuration file
+    allows to convert the system without any issue.
+    The reference ticket: https://issues.redhat.com/browse/RHELC-1180
+    """
+    if not os.path.exists(FIREWALLD_CONFIG_FILE):
+        pytest.fail("Firewalld configuration file does not exist")
+
+    assert shell("systemctl start firewalld").returncode == 0
+    assert shell("systemctl enable firewalld").returncode == 0
+
+    assert (
+        shell(f"sed -i 's/CleanupModulesOnExit=yes/CleanupModulesOnExit=no/g' {FIREWALLD_CONFIG_FILE}").returncode == 0
+    )
```

Comment:
This will fail if we fail to match `CleanupModulesOnExit=yes`.
We might consider the suggestion from the other (non-destructive) test case. To provide and copy over our own config file and back up the existing one.